### ISSUE TITLE
Failed to connect using SSL

### ISSUE THREAD

**Issue by oywino** [axzae/homeassist#25]
The headline says it all. I connect remotely to HA using Chrome over SSL which works just fine, both from a PC add all as from my Android phone. But the app failed after I upgraded from HTTP to HTTPS.
The error message just says that the WS could not be established.

**Comment by payano** [comment]
Hello @oywino, is this still a problem? I am using https on my home assistant installation.
Are you using a self signed certificate or a certificate signed by a trusted authority?

**Comment by oywino** [comment]
Yes, this is still a problem, but only while using the Android App. But the problem has worsened after upgrading HA to the latest version as the app doesn't even have an option to enter user name. It only accepts URL + password which is insufficient to log on to HA.

**Comment by payano** [comment]
URL + password should be sufficient for the HA when using websockets, two authentication methods is available: "access_token" and "api_password" (legacy auth) [1].
In order to test this i need to reproduce the error somehow, could you open a new ticket in my forked repository and provide me with an configuration file that produces this behaviour?
https://github.com/payano/homeassist-ng/

[1] https://developers.home-assistant.io/docs/en/external_api_websocket.html
